tests/integration/test_bigquery.py

```diff
@@ -1,48 +1,86 @@
+from typing import List
 import pandas as pd
+import pandas_gbq
+import pytest
+from numpy import ndarray
 
+from viadot.exceptions import CredentialError, DBDataAccessError
 from viadot.sources import BigQuery
 
-BIGQ = BigQuery(credentials_key="BIGQUERY_TESTS")
 
+@pytest.fixture(scope="function")
+def BIGQ():
```

Comment:
Please add docstring.

def BIGQ():
    """
        Fixture for creating a BigQuery client instance.
        This fixture initializes a BigQuery client using the provided credentials key and yields 
        the client instance.
        The client instance can be used within a test function to interact with BigQuery.
        Returns:
        BigQuery: A BigQuery client instance
    """